### ISSUE TITLE
About the data set

### ISSUE THREAD

**Issue by aaawrong** [WeichuangLi/GASENet#1]
Hello, thanks for your code and I would like to ask why we divide the picture of 1024x124 into 16 copies of 256x256. Can we take a screenshot of the face with 256x256?

**Comment by WeichuangLi** [comment]
Thanks for your attention.

The reason is two-fold:
(i) We can deprecate the semantics in images, which is usually unwanted for the task of generated image detection;
(ii) The cropped patches will not cripple the artifacts of generated images. 

During inference, you can certainly try this for 256x256 faces (actually StarGAN and RelGAN results are tested with full images). However, we recommend you to use cropped patches to get better robustness.

**Comment by DCMJY** [comment]
thanks you very much : )

**Comment by DCMJY** [comment]
HI,I'm sorry to bother you again.
When I resize 1024 images to 256 as the data set input, the result is very poor.Do you have any idea?

**Comment by DCMJY** [comment]
I'm sorry to bother you again.
I saw that there are different kinds of StarGAN in the eval.py but there is only one kind of data in the paper.Do you average the different kinds of StarGAN as the paper data?

**Comment by WeichuangLi** [comment]
> I'm sorry to bother you again. I saw that there are different kinds of StarGAN in the eval.py but there is only one kind of data in the paper.Do you average the different kinds of StarGAN as the paper data?

Yes, the reported result is averaged over these different kinds of images. The "StarGAN-short" is just for code testing. Moreover, the result of "Transformer" (also known as "VQ-GAN") is also averaged over the result of "FFHQ" and "CelebA-HQ."

**Comment by DCMJY** [comment]
Thank you for your answer.
But the result of "Transformer" I test only have about 80% acc. Do you have any idea?
I downloaded "Transformer" datasets  from the office code "https://github.com/CompVis/taming-transformers" and I use the early stop.

**Comment by WeichuangLi** [comment]
It is after submission that we realize the VQ-GAN is not a conventional GAN. It is an auto-regressive-based method indeed. Therefore, the performance may not be that good. Personally, I think that 80% accuracy is a satisfactory result. 

Even though we tried to decouple the semantic information via cropping, it cannot be fully disentangled. It is recommended to test the image with faces. 
Also, the performance depends on the images(the training and testing dataset). Different images or even different patches might contribute differently. You might try different patches as well.

**Comment by DCMJY** [comment]
Ohhhhh , thank you for your quick and detailed answer : )

**Comment by DCMJY** [comment]
Hi,sorry sorry to bother you again.
In the paper "For baselines, positive and negative samples from different domains are mixed up to conduct training" mean mix celebaHQ FFHQ as real ,mix pggan stylegan as fake? Or celebaHQ  pggan as a group to train once and FFHQ stylegan to train once,then take average.

**Comment by TrevorPatrick0918** [comment]
We chose the former strategy. The latter might be unfair, as the amount of training dataset is different.

**Comment by WeichuangLi** [comment]
We chose the former strategy. The latter might be unfair, as the amount of training dataset is different. Also, deploying the latter strategy will not have a good intra-class performance.
